style.css

```diff
@@ -535,3 +534,302 @@ div.item .langu {
   height: 5px;
   margin-left: 120px;
 }
+
+/* Desktop Version Start */
+
+@media screen and (min-width: 768px) {
+  .navbar {
+    background: #3c3a39;
+    width: 100%;
+    height: 92px;
+  }
+
```

Comment:
- [ ] According to the project requirement you should make sure that the  Desktop version looks like the one in the design you chose, I can see that your website is not really responsive while switching from mobile to desktop looks 👇🏻 

![127 0 0 1_5501_ (1)](https://user-images.githubusercontent.com/79058364/167035136-cd7bee22-de1d-4d8d-b913-56cb99dbd757.png)